core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -158,10 +158,11 @@ private[log] class LogCleanerManager(val logDirs: Array[File], val logs: Pool[To
           inProgress.put(topicPartition, LogCleaningPaused)
         case Some(state) =>
           state match {
+            case LogCleaningPaused => {}
```

Comment:
I had a quick look and found https://issues.apache.org/jira/browse/KAFKA-3978

It's not exactly the same stack (in there it's coming from ReplicaManager) but it also blows up in Log.truncateTo with the same exception